src/System.Data.SQLite/NativeMethods.cs

```diff
@@ -127,6 +127,56 @@ internal static class NativeMethods
 		[DllImport(c_dllName, CallingConvention = c_callingConvention)]
 		public static extern int sqlite3_trace_v2(SqliteDatabaseHandle db, SQLiteTraceEvents eventsMask, SQLiteTraceV2Callback callback, IntPtr userData);
 
+#if NET5_0
```

Comment:
```suggestion
#if NET5_0_OR_GREATER
```

Reply:
This doesn't work, at least on my machine using VS2022.

Reply:
Okay, I guess we're stuck with `NET5_0`. I wonder what enables it...

Reply:
I think it was first supported in the .NET 6 SDK.

Reply:
https://github.com/dotnet/designs/blob/main/accepted/2020/or-greater-defines/or-greater-defines.md says ".NET 6", https://github.com/dotnet/docs/blob/main/includes/preprocessor-symbols.md says ".NET 5+ SDK".